simplesat/constraints/tests/test_requirement.py

```diff
@@ -144,7 +144,18 @@ def test_any(self):
         self.assertTrue(requirement.matches(V("1.9.0-1")))
         self.assertEqual(requirement, r_requirement)
         self.assertEqual(requirement, r_requirement_empty)
+        self.assertEqual(r_requirement, r_requirement_empty)
+
+        # Given
+        requirement_string = "numpy *"
```

Comment:
I actually thought this was invalid for the preamble, but I suppose this makes sense. Is the thought that someone might type this on the command line?  `jaguar install 'numpy *'` ?